runtime/gc_vlhgc/RememberedSetCardBucket.cpp

```diff
@@ -184,10 +184,11 @@ MM_RememberedSetCardBucket::getSize(MM_EnvironmentVLHGC *env)
 	UDATA size = _bufferCount * MAX_BUFFER_SIZE;
 	if (_bufferCount > 0) {
 		Assert_MM_true(NULL != _current);
-		UDATA offset = (UDATA)_current & OFFSET_MASK;
+		UDATA offset = (UDATA)_current & offsetMask(env);
 		if (0 != offset) {
 			/* subtract the unused portion of the current buffer */
-			size -= MAX_BUFFER_SIZE - (offset / sizeof (MM_RememberedSetCard));
+			uintptr_t const referenceSize = env->compressObjectReferences() ? sizeof(uint32_t) : sizeof(uintptr_t);
```

Comment:
I said this before, but there should be a helper like EnvBase::getReferenceSize(). You've just introduced similar one for RS Card size. But this being an OpenJ9 change and EnvBase being in OMR, of course, this is not the right moment to introduce it.

Reply:
as mentioned before, we are dealing with card size, not  reference size

note that in future, due to 9 bit shifting (and all the work you've done), we may end up compressing RS card for nonCR object references up to certain heap size (2^card shift * 4GB).

Reply:
Done.